src/events.js

```diff
@@ -880,6 +854,58 @@ if (confirmVal === "cancel") {
         searchTemplate();
     }
 }
+    
+
+const showConfirmationModal = async (collectionID, firstName) => {
```

Comment:
To me, there's no need to wrap this function as an async function. You can refactor it if you have time.

Reply:
Is the reason for not wrapping this function in async because `showConfirmationModal` returns a promise explicitly? Is the async keyword that ensures a promise a return redundant?